ops/terraform/env/prod/stateful/main.tf

```diff
@@ -10,11 +10,15 @@ provider "aws" {
 module "stateful" {
   source = "../../../modules/stateful"
 
-  # Large DB
   db_config = { 
     instance_class    = "db.r5.24xlarge"
-    iops              = 5000
-    allocated_storage = 10000
+    iops              = 16000
+    allocated_storage = 16000
```

Comment:
I've had issues where the iops/storage ratio wasn't accepted. I'd be curious if this will work.